### PR TITLE
Update project-goals.md

### PR DESCRIPTION
**Pull request by quirico**
Fix spelling

### CODE REVIEW

**Review by marvinhagemeister — APPROVED** [review]
Thanks for the PR 👍

File: content/en/about/project-goals.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Preact aims to deliver on a few key goals:
 
 - **Performance:** Render quickly & efficiently
 - **Size:** Small size, lightweight _(approximately 3.5 kB)_
-- **Efficiency:** Effective memory usage _(avoiding GC thrash)_
+- **Efficiency:** Effective memory usage _(avoiding garbage collection trash)_
```
Comment (rschristian) [inline content/en/about/project-goals.md:15]: Thrash is the right word here. "avoiding garbage collection trash" doesn't make much sense.

Comment (quirico) [reply content/en/about/project-goals.md:15]: Sorry for this, not being a native speaker makes difference here. May I say that it still remains unclear to me, indeed?

Comment (rschristian) [reply content/en/about/project-goals.md:15]: It's just something you'll need to learn then.

Thrashing, especially in relation to GC, is terminology in this field. While "thrash" may be a less common word, it's the correct one for this use case. See: https://en.wikipedia.org/wiki/Thrashing_(computer_science)

Comment (quirico) [reply content/en/about/project-goals.md:15]: Right, @rschristian, thank you for the explanation and the link! :)

**Pull request merged**